app/scripts/store/dimStores.directive.js

```diff
@@ -29,7 +29,7 @@
       scope: {},
       link: Link,
       template: [
-        '<div ng-if="vm.stores.length" ng-class="[\'dim-col-\' + vm.settings.charCol, { \'hide-filtered\': vm.settings.hideFilteredItems, itemQuality: vm.settings.itemQuality }]">',
+        '<div ng-if="vm.stores.length" ng-class="[\'dim-col-\' + vm.settings.charCol, { \'hide-filtered\': vm.settings.hideFilteredItems }]">',
```

Comment:
I thought about it! I'm a bit worried that it'd cause items in loadouts/xur to disappear when there's an active filter... and I didn't bother to test that theory.